python/test/unit/language/test_core.py

```diff
@@ -2263,7 +2272,18 @@ def test_scan_layouts(M, N, src_layout, axis, device):
     np.testing.assert_equal(z_ref, z_tri.cpu().numpy())
 
 
-layouts = [
+if is_hip():
+    layouts = [
+    MfmaLayout(version=(2,0), warps_per_cta=[4, 1], instr_shape=[32,32], is_transposed=True),
```

Comment:
Since this is the first time we add MfmaLayout, can we also grab the version from the driver, so that we can also test it on MI300?

Reply:
Done.